### PR TITLE
Add link to poisson_1d_pideeponet example.

### PR DESCRIPTION
**Pull request by samuelburbulla**

### CODE REVIEW

**Comment by lululxvi** [comment]
The docs doesn't work well https://deepxde--1462.org.readthedocs.build/en/1462/demos/operator/poisson_1d_pideeponet.html

**Comment by samuelburbulla** [comment]
Sorry for this oversight! Fixed the file names.

**Comment by lululxvi** [comment]
Change file name to poisson.1d.pideeponet.rst

File: docs/demos/operator.rst [hunk 1]
```diff
@@ -15,6 +15,12 @@ DeepONet
 PI-DeepONet
 -----------
 
+.. toctree::
+   :maxdepth: 1
+
+   operator/poisson_1d_pideeponet
```
Comment (lululxvi) [inline docs/demos/operator.rst:21]: Should this line also change?

**Pull request merged**